File: proyecto_python/paquete03/atletismo.py
def registrar_participante_atletismo():
    nombre = input("Ingrese el nombre del participante: ")
    edad = int(input("Ingrese la edad del participante: "))
    especialidad = input("Ingrese la especialidad (carreras, salto, lanzamiento): ")
    ciudad = input("Ingrese la ciudad: ")
    record_personal = input("Ingrese el récord personal: ")

    cadena = f"""
    Resumen de registro - Atletismo:
    Nombre: {nombre}
    Edad: {edad}
    Especialidad: {especialidad}
    Ciudad: {ciudad}
    Record Personal: {record_personal}
    """

    return cadena

File: proyecto_java/Problema01/src/paquete03/Atletismo.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package paquete03;

import java.util.Scanner;

/**
 *
 * @author Core i5-Pro
 */
public class Atletismo {
     public static String registrarParticipanteAtletismo() {
        Scanner entrada = new Scanner(System.in);
        String mensaje;
        String nombre;
        int edad;
        String especialidad;
        String ciudad;
        String recordPersonal;
        System.out.print("Ingrese el nombre del participante: ");
        nombre = entrada.nextLine();
        System.out.print("Ingrese la edad del participante: ");
        edad = entrada.nextInt();
        entrada.nextLine();
        System.out.print("Ingrese la especialidad (carreras, salto, lanzamiento)"
                + ": ");
        especialidad = entrada.nextLine();
        System.out.print("Ingrese la ciudad: ");
        ciudad = entrada.nextLine();
        System.out.print("Ingrese el récord personal: ");
        recordPersonal = entrada.nextLine();
        
        String cadena = "Resumen de registro - Yoga:\n" +
               "Nombre: " + nombre + "\n" +
               "Edad: " + edad + "\n" +
               "Especialidad: " + especialidad + "\n" +
               "Ciudad: " + ciudad + "\n" +
               "Record Personal: " + recordPersonal + "\n";
        mensaje= cadena;

        return mensaje;
    }
}
